docs/source/algorithms/SingleCrystalDiffuseReduction-v1.rst

```diff
@@ -61,15 +62,14 @@ If no background is used then the "ws" is just the normalised data.
 Symmetries
 ##########
 
-The symmetry is applied by manipulating the :ref:`UB matrix <Lattice>`.
-The SymmetryOps parameters can either be defined as a :ref:`space group <Point and
-space groups>` (number or name) were all the symmetries for that group
-is applied, or you can specify individual :ref:`symmetries <Symmetry
-groups>` to apply.
+The SymmetryOperations parameters can either be defined as a
+:ref:`point or space group <Point and space groups>` were all the
+symmetries for that group is applied, or you can specify individual
+:ref:`symmetries <Symmetry groups>` to apply.
 
-For example setting SymmetryOps to "P 31 2 1", "152" or "x,y,z;
--y,x-y,z+1/3; -x+y,-x,z+2/3; y,x,-z; x-y,-y,-z+2/3; -x,-x+y,-z+1/3"
-are equivalent.
+For example setting SymmetryOperations to space group "P 31 2 1",
+point group "321" or "x,y,z; -y,x-y,z; -x+y,-x,z; y,x,-z; x-y,-y,-z;
+-x,-x+y,-z" are equivalent.
```

Comment:
Maybe I should have added to mine as well, but to clarify any confusion, these are symmetries of the real lattice, not of the reciprocal lattice.